packages/perseus/src/widgets/radio/radio.class.new.tsx

```diff
@@ -21,6 +21,7 @@ export type RenderProps = {
     countChoices?: boolean;
```

Comment:
I've deleted this file in my restructure PR, but I'll happily pull this work over into that PR before landing. (I'll let you land first). I'll send you a message once I've integrated your changes, just so that you can review to verify that I didn't miss anything. :)